kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/AdminClientWrapper.java

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
So I am not sure what the value of this wrapper is. It seems to just duplicate functionality in the KafkaAdminClient which itself is a wrapper for the Kafka AdminClient. If you are having the wrapper, I would have thought it would have been more useful turning the async code into promise based futures so the methods return a Vertx Future to allow you to use the `.onSuccess`, `.onFailure`, `.onComplete` methods on the `Future`. Vertx 4 does this in the Vertx KafkaClient but it is not GA yet. From experience, I am also nervous throwing exceptions in a Vertx environment because Vertx has a tendency to swallow exceptions and it is not obvious where things end up. It is worse than a `goto` in my view.